scm/pipeline.go

```diff
@@ -0,0 +1,106 @@
+package scm
+
+type (
+	ObjectAttributes struct {
+		ID         int
+		IID        int
+		Name       string
+		Ref        string
+		Tag        bool
+		SHA        string
+		BeforeSHA  string
+		Source     string
+		Status     string
+		Stages     []string
+		CreatedAt  string
+		FinishedAt string
+		Duration   int
+		Variables  []Variable
+		URL        string
+	}
+
+	Variable struct {
+		Key   string
+		Value string
+	}
+
+	MergeRequest struct {
+		ID                  int
+		IID                 int
+		Title               string
+		SourceBranch        string
+		SourceProjectID     int
+		TargetBranch        string
+		TargetProjectID     int
+		State               string
+		MergeStatus         string
+		DetailedMergeStatus string
+		URL                 string
+	}
+
+	Project struct {
```

Comment:
`Project` is specific to Harness.  We abstract using the `Organization` structure

Reply:
Removed Project